straxen/corrections_services.py

```diff
@@ -35,12 +36,12 @@ def __init__(self, username='nt_analysis', password=None, is_nt=True):
         if password is not None:
             self.password = password
         elif self.username.endswith('analysis'):
-            self.password = straxen.get_secret('rundb_password')
+            self.password = uconfig.get('RunDB', 'pymongo_password')
         else:
             raise ValueError(f'No password for {username}')
 
         # Get the readonly account for the rundb using hostname = ''
-        runsdb_mongo_url = straxen.rundb.get_mongo_url(hostname='')
+        runsdb_mongo_url = uconfig.get('RunDB', 'pymongo_url')
```

Comment:
this should also be more flexible, bootstrax may be running with admin privileges but I would not want to be running with those here.

Reply:
Yeah so the bootstrax config would look different -- or is this not the functionality you want? In my view this should be handled by a config file, not hardcoded. Maybe I'm missing something though.

Reply:
So perhaps we can find a solution. So it seems to me that in principle, not all these usages of `pymongo_url` have to be the same. Like in bootstrax I want to have write permissions in https://github.com/XENONnT/straxen/blob/ca8dc1af05dbab89c8cbb2a4f14ad5bd5ba74bda/straxen/online_monitor.py#L29 while having readonly access here. 

Like this, just specifying the `pymongo_database` in the config is not able to distinguish right? You will load all with the same authentication/privilages. One could argue that one with write access should be comfortable doing everything with write access but I'm just trying to see if we can minimize things that we know one should not touch (if that makes sense).


There are some things possible, let me know what you think is best / you have a better idea:
 - Create `pymongo_database_RunDB`, `pymongo_database_CMT`, `pymongo_database_...` in the ini file, write those at the appropriate places. The ini file would be specifying the same things under a different name many times but at least it would be super transparent.
 - Shift all this to where one loads these different classes like for instance here: https://github.com/XENONnT/straxen/blob/master/straxen/contexts.py#L103 to ask for a different password/username and database. I'm not sure if it's transparent though.
- Create an `pymongo_admin_username`, `pymongo_admin_password`, `pymongo_admin_url`, `pymongo_admin_database` in the ini file. Then, using `uconfig.get` one should explicitly write down when trying to start any of these classes with the write access. This means changing this line to `if runsdb_mongo_url is None: ....` and allowing one to specify it in the init of this class. 

I think the last options seems to make most sense. Sorry for the long story here.

Reply:
Just to recap @jorana and my conversation offline, we will have bootstrax use a different configuration parameter to prevent unwanted db calls on the EBs with admin privileges.